libvirt/tests/src/virtual_disks/virtual_disks_throttlefilters.py

```
@@ -0,0 +1,188 @@
+    throttlefilter_group2 = params.get("throttlefilter_group2")
+    throttle_options = params.get("throttle_options")
+    dd_count = params.get("dd_count")
+    check_qemu_pattern = params.get("check_qemu_pattern")
+    server_user = params.get("server_user")
+    server_pwd = params.get("private_key_password")
+    mon_host = params.get("mon_host")
+
+    vmxml = vm_xml.VMXML.new_from_inactive_dumpxml(vm_name)
+    vmxml_backup = vmxml.copy()
+    disk_obj = disk_base.DiskBase(test, vm, params)
+    file_list = []
```

Ruff (0.14.10)

Local variable `file_list` is assigned to but never used

Remove assignment to unused variable `file_list`

(F841)